### PR TITLE
Adding in color name setting

### PR DESCRIPTION
**Pull request by crutkas**
This setting wasn't included in the doc

### CODE REVIEW

**Comment by PRMerger19** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Court72** [comment]
@mattwojo 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"

File: hub/powertoys/color-picker.md [hunk 1]
```diff
@@ -62,6 +62,8 @@ Color Picker has following settings:
 
 - Activation shortcut
 - Behavior of activation shortcut
+- Show the name of the color.
+   - The shown color name will be a high-level representation of the color. Example: Light Green, Green, Dark Green.
```
Comment (htcfreek) [inline hub/powertoys/color-picker.md]: has to be switched with the copy format (see order on screenshot)

**Review by htcfreek — CHANGES_REQUESTED** [review]

File: hub/powertoys/color-picker.md [hunk 2]
```diff
@@ -62,6 +62,8 @@ Color Picker has following settings:
...
 - Format of a copied color (HEX, RGB, etc.)
```
Comment (Jay-o-Way) [inline hub/powertoys/color-picker.md]:
```suggestion
- Format of a copied color (HEX, RGB, etc.)
- Show the name of the color. The shown color name will be a high-level representation of the color. Example: Light Green, Green, Dark Green.
```

**Comment by PRMerger17** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by mattwojo — APPROVED** [review]

**Comment by mattwojo** [comment]
#sign-off

**Review by htcfreek — APPROVED** [review]
LGTM

**Pull request merged**